File: Cpp_solving/exercise_05.cpp
/*
    Given two distinct integer values, issue an appropriate legend that reports 
    which is the greatest among them.
*/

#include <iostream>
using namespace std;

// I create the resolving function
int show_greatest(int value_1, int value_2){
    if (value_1 > value_2) return value_1;
    else return value_2;
}

// Showing the greatest in main()
int main () {
    int result = show_greatest(10,20);
    cout << "The gratest number is: " << result << endl;
 return 0;
}

File: Python_solving/exercise_05.py
#   Given two distinct integer values, issue an appropriate legend that reports 
#   which is the greatest among them.

# I create a lambda function that's gonna show the greatest among the two values.
whos_greatest = lambda a,b: print("A is greatest than B.") if a > b else print("B is greatest than A.")
# Showing result
whos_greatest(10,22)